lib/Jsrt/ChakraCore.h

```diff
@@ -42,66 +48,115 @@ typedef void* JsModuleRecord;
 /// </remarks>
 typedef void *JsSharedArrayBufferContentHandle;
 
+/// <summary>
+///    Flags for parsing a module.
+/// </summary>
 typedef enum JsParseModuleSourceFlags
 {
+    /// <summary>
+    /// Module source is UTF16.
+    /// </summary>
     JsParseModuleSourceFlags_DataIsUTF16LE = 0x00000000,
+    /// <summary>
+    /// Module source is UTF8.
+    /// </summary>
     JsParseModuleSourceFlags_DataIsUTF8 = 0x00000001
 } JsParseModuleSourceFlags;
 
+/// <summary>
+///    The types of host info that can be set on a module record with JsSetModuleHostInfo.
+/// </summary>
+/// <remarks>
+/// For more information see JsSetModuleHostInfo.
+/// </remarks>
 typedef enum JsModuleHostInfoKind
 {
+    /// <summary>
+    /// An exception object - e.g. if the module file cannot be found.
+    /// </summary>
     JsModuleHostInfo_Exception = 0x01,
+    /// <summary>
+    /// Host defined info.
+    /// </summary>
     JsModuleHostInfo_HostDefined = 0x02,
+    /// <summary>
+    /// Callback for receiving notification when module is ready.
+    /// </summary>
     JsModuleHostInfo_NotifyModuleReadyCallback = 0x3,
+    /// <summary>
+    /// Callback for receiving notification to fetch a dependent module.
+    /// </summary>
     JsModuleHostInfo_FetchImportedModuleCallback = 0x4,
+    /// <summary>
+    /// Callback for receiving notification for calls to ```import()```
+    /// </summary>
     JsModuleHostInfo_FetchImportedModuleFromScriptCallback = 0x5,
+    /// <summary>
+    /// URL for use in error stack traces and debugging.
+    /// </summary>
     JsModuleHostInfo_Url = 0x6
 } JsModuleHostInfoKind;
 
 /// <summary>
 ///     User implemented callback to fetch additional imported modules in ES modules.
 /// </summary>
 /// <remarks>
-/// Notify the host to fetch the dependent module. This is the "import" part before HostResolveImportedModule in ES6 spec.
-/// This notifies the host that the referencing module has the specified module dependency, and the host need to retrieve the module back.
+/// The callback is invoked on the current runtime execution thread, therefore execution is blocked until 
+/// the callback completes. Notify the host to fetch the dependent module. This is the "import" part 
```

Comment:
I know this block was not indented previously, but could you add an indent to align with the rest of the file?